### ISSUE TITLE
[Bug] Cannot decrypt tweet when user post multi magic url

### ISSUE THREAD

**Issue by Tedko** [DimensionDev/Maskbook#1095]
# Bug Report

<img width="591" alt="Screen Shot 2020-05-16 at 03 21 45" src="https://user-images.githubusercontent.com/6687437/82090645-87242380-9728-11ea-8e4d-a01c59271341.png">

Cannot decrypt this post (also say Maskbook decrypting... forever)

### System

- OS:
- OS Version:

### Browser

- Browser:
- Browser Version:

### Maskbook

- Maskbook Version:
- Installation: /* May be "Store", "ZIP", or "Self-Complied" */
- Build Commit: Optionally attach a Commit ID, if it is from an pre-release branch head

### Expected Behavior

/* Write the expected behavior here. */

### Actual Behavior

/* Write the actual behavior here. */

### How To Reproduce

/* Specify how it may be produced here. */

**Comment by Tedko** [comment]
```
Unknown
Unknown
-38
7fQMhDZX3nUOHYUuf9C6eg/UZYtjD/7ybNEl4+xhEkFUO5meIEkGuQh+4GeTrRIAQ2kR9UmkyQX/SVKGluvyhAB1QQflbfBITFebdAybIx7fNppXWFQIGEimGi1Iu2TdJkmlT9nVA9i0jfPRstD0QVQBEqVyNNUfl/pqM7xKN5XeS1rP7zssnptA
nl9wHvvntRQun2XL+Fq4PA==
ZctmQ1wetWHjBWyeKUSfsFyjfnX8h2SpVCbymjADkh2LvjkeRitLG94=
eVaP1djyPFqDmVrS+Wlj25VeGODDRIotW26tPL4ad7OgDuCMobweB5DcxS53u2hlBXdd0r4pqYk3e+CIdNJ3/w==
Nickname Suji Yan, UserID suji_yan
Ao+TblKN40/JUXl4B1HsISVoJc5gSVBYCXiolhxa8D5Q
post:1260194173346447367/person:twitter.com/
http://maskbook.com https://www.wordpress.com/%204/4.7fQMhDZX3nUOHYUuf9C6eg/UZYtjD/7ybNEl4-xhEkFUO5meIEkGuQh+4GeTrRIAQ2kR9UmkyQX/SVKGluvyhAB1QQflbfBITFebdAybIx7fNppXWFQIGEimGi1Iu2TdJkmlT9nVA9i0jfPRstD0QVQBEqVyNNUfl/pqM7xKN5XeS1rP7zssnptA.nl9wHvvntRQun2XL+Fq4PA_=.ZctmQ1wetWHjBWyeKUSfsFyjfnX8h2SpVCbymjADkh2LvjkeRitLG94=.eVaP1djyPFqDmVrS+Wlj25VeGODDRIotW26tPL4ad7OgDuCMobweB5DcxS53u2hlBXdd0r4pqYk3e+CIdNJ3/w==.AsLGWRmbL0C/ejYvSOHn22bEt7PMwYnLGbij84GZEhe6.0%40 #Maskbook ([I:b]) 使用 http:// @realMaskbook 解密这条推文。 —§— /* https:// X3nUOHYUuf9C6eg/UZYtjD/7ybNEl4-xhEkFUO5meIEkGuQh+4GeTrRIAQ2kR9UmkyQX/SVKGluvyhAB1QQflbfBITFebdAybIx7fNppXWFQIGEimGi1Iu2TdJkmlT9nVA9i0jfPRstD0QVQBEqVyNNUfl/pqM7xKN5XeS1rP7zssnptA.nl9wHvvntRQun2XL+Fq4PA_=.ZctmQ1wetWHjBWyeKUSfsFyjfnX8h2SpVCbymjADkh2LvjkeRitLG94=.eVaP1djyPFqDmVrS+Wlj25VeGODDRIotW26tPL4ad7OgDuCMobweB5DcxS53u2hlBXdd0r4pqYk3e+CIdNJ3/w==.AsLGWRmbL0C/ejYvSOHn22bEt7PMwYnLGbij84GZEhe6.0%40 … */
```

**Comment by neruthes** [comment]
I did not foresee that this undefined behavior would be exploited such early.

Maybe we want to accept the first Magic URL at this moment. In the long run, we will need to find an optimal solution.

**Comment by Jack-Works** [comment]
this might be closed by #1344

**Comment by guanbinrui** [comment]
> this might be closed by #1344

No, the user should post with more than one type of payload.
